TypeCobol.Analysis/Report/ZCallPgmReport.cs

```diff
@@ -0,0 +1,312 @@
+        private void OnUsePointCreated(object sender, DfaUsePoint<Node, VariableSymbol> up)
+        {
+            switch (up.Instruction.CodeElement.Type)
+            {
+                case CodeElementType.CallStatement:
+                    {
+                        CallStatement callStatement = up.Instruction.CodeElement as CallStatement;
+                        string pgmVar = callStatement?.InputParameters.First().StorageAreaOrValue.MainSymbolReference.Name;
+                        if (up.Variable.Name.Equals(pgmVar, StringComparison.InvariantCultureIgnoreCase))
+                        {
+                            var target = up.Instruction.CodeElement.CallSites.First().CallTarget;
+                            foreach (var callStyle in _AlternativeCallList)
+                            {
+                                if (target != null && target.ToString().Equals(callStyle, StringComparison.OrdinalIgnoreCase))
+                                {
+                                    _callUsePoints.Add(up);
+                                    if (up.Variable.Type.Tag == Compiler.Types.Type.Tags.Group)
+                                    {
+                                        //Compute the Map of Level88 variable to their parent..
+                                        if (Level88SymbolParentSymbolMap == null)
+                                        {
+                                            Level88SymbolParentSymbolMap = new Dictionary<VariableSymbol, VariableSymbol>();
+                                        }
+                                        Level88SymbolCollector l88c = new Level88SymbolCollector(up.Variable);
+                                        up.Variable.Accept(l88c, Level88SymbolParentSymbolMap);
+                                    }
+                                    break;
+                                }
+                            }
+                        }
+                    }
+                    break;
+            }
+        }
+
+        /// <summary>
+        /// Called when a DefPoint is built. If the Def point is a level 88 variable of the target CALL variable
+        /// then we deceive DFA algorithm to think that the target variable of the DefPoint is the CALL variable
+        /// thus we change the target variable of the DefPoint.
+        /// </summary>
+        /// <param name="sender">The sender is the Dfa Builder</param>
+        /// <param name="dp">The Def Point</param>
+        private void OnDefPointCreated(object sender, DfaDefPoint<Node, VariableSymbol> dp)
+        {
+            VariableSymbol sym = dp.Variable;
+            if (sym.Level == 88 && sym.Value != null && Level88SymbolParentSymbolMap != null && Level88SymbolParentSymbolMap.TryGetValue(sym, out var parent))
+            {//Now say that it is the parent variable which is the target of the definition.
+                dp.Variable = parent;
+                dp.UserData = sym;
+            }
+        }
```

Comment:
As mentioned in #1751, this is a 'hack' that should be generalized into `DefaultDataFlowGraphBuilder` but the problem is bigger. In fact we have the same problem for every group variable. If any child of a group is modified, then it means for DFA that the whole group is modified. Here we handle modification of a level 88 and we use this info only when it is used in a CALL.

Reply:
I resolve the conversation, this would be addressed with #1807.